.github/workflows/ci-build-react-native-test-v1.yml

```diff
@@ -11,7 +11,7 @@ jobs:
     runs-on: ubuntu-latest
     strategy:
       matrix:
-        node-version: [ 12.x ]
+        node-version: [ 14.x ]
```

Comment:
Thank for updating this. Maybe we should go to lowest active e.g. 16?

Reply:
Based on https://endoflife.date/nodejs and https://github.com/nodejs/release#release-schedule lowest `Active Support` version is 18,  lowest `Security Support` is 14.